plugins/shortcuts/mpris.js

```diff
@@ -94,40 +84,48 @@ function registerMPRIS(win) {
 		player.on('seek', seekBy);
 		player.on('position', seekTo);
 
+		player.on('shuffle', (enableShuffle) => {
+			shuffle();
+		});
+
 		ipcMain.on('volumeChanged', (_, value) => {
-			player.volume = value;
+			if (config.plugins.isEnabled('precise-volume')) {
```

Comment:
what is this about? `precise-volume` doesn't change the way the volume is transmitted?

https://github.com/th-ch/youtube-music/blob/0f09f8a8ed8d6ffe5e95aaf47fce7368800bbdf4/providers/song-info-front.js#L79-L85

Reply:
Yes, I must have missed that from when I was debuging. I will remove the if.